### PR TITLE
P-2523 Swiper component

### PR DESCRIPTION
**Pull request by gunjankothari**

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -139,7 +142,7 @@ function Examples() {
   return (
     <NavigationContainer>
       <Drawer.Navigator
-        initialRouteName="CircleImage"
+        initialRouteName="Swiper"
```
Comment (dhruska) [inline example/src/App.tsx:145]:
```suggestion
        initialRouteName="CircleImage"
```

File: packages/core/src/components/Swiper/SwiperItem.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { COMPONENT_TYPES } from "@draftbit/types";
+import React from "react";
+import { View, StyleSheet, StyleProp, ViewStyle } from "react-native";
+
+const styles = StyleSheet.create({
+  wrapper: {
+    flex: 1,
+    alignItems: "center",
+    justifyContent: "center",
+  },
```
Comment (dhruska) [inline packages/core/src/components/Swiper/SwiperItem.tsx:10]: Should these be user-configurable? If so we should put them in the `layout` object in `SEED_DATA`. If not, disregard

**Comment by dhruska** [comment]
Just one question - this looks good, nice work!

File: packages/core/src/components/Swiper/SwiperItem.tsx (hunk 2 above)
Comment (gunjankothari) [reply packages/core/src/components/Swiper/SwiperItem.tsx]: Actually, It is not required here, moved it to the example.

**Review by dhruska — APPROVED** [review]

**Pull request merged**